### DISCUSSION TITLE
Cumulative flow diagram

### DISCUSSION THREAD

**Discussion by kfurgerot** [nbrown02/FlowViz-Jira#26] [Ideas]
Hi,

It will be very interesting to add a cumulative flow diagram to the powerbi dashboard.
It will show us how stable is the flow. 
It displays any weaknesses and provides us with actionable advice for improvement.

**Comment by nbrown02** [comment]
I don't have any plans to build a CFD - I find them to be overrated and you'll get 5 different explanations of them if you talk to 5 different Agile Coaches :D

**Reply by kfurgerot** [reply]
So sad :) but i understand
